pkg/grpc/authn/extractor_test.go

```diff
@@ -0,0 +1,65 @@
+package authn
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/grpc/requestinfo"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/square/go-jose.v2/jwt"
+)
+
+func TestExtractorErrorNilGuards(t *testing.T) {
+	var err *ExtractorError
+
+	assert.Equal(t, "", err.Error())
+	assert.Nil(t, err.Unwrap())
+	assert.NotPanics(t, func() {
+		err.LogL(requestinfo.RequestInfo{})
+	})
+}
+
+func TestExtractorErrorUnwrap(t *testing.T) {
+	jwtErr := jwt.ErrExpired
+	err := NewExtractorError("test", "test msg", jwtErr)
+	assert.ErrorIs(t, err.Unwrap(), jwt.ErrExpired)
```

Comment:
If you unwrap then we should check for equality, with errorIs there is no reason to unwrap
```suggestion
	assert.Equal(t, err.Unwrap(), jwt.ErrExpired)
	assert.ErrorIs(t, err, jwt.ErrExpired)
```